cypress/support/commands.js

```diff
@@ -3,7 +3,7 @@ Cypress.Commands.add('requestGraphQl', operationName => {
         url: '/',
         method: 'POST',
         headers: {
-            'Authorization': '',
+            'Authorization': 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjIyMDgsInVzZXJUeXBlSWQiOjYsImJ1eWVyVHlwZSI6MiwiY291bnRyeSI6eyJpZCI6MTkzLCJuYW1lIjoiU0ciLCJkZXNjcmlwdGlvbiI6IlNpbmdhcG9yZSIsImN1cnJlbmN5X2NvZGUiOiJTR0QiLCJjdXJyZW5jeV9zeW1ib2wiOiJTJCIsImFjdGl2ZSI6dHJ1ZX0sImlhdCI6MTYyMTI0MDE5MX0.fgFzpKDjrRkUKko9-JncZxkyDvOKe5KL0Lrh7fF93PI',
```

Comment:
Remove this

Reply:
Ini ndak ke handle gitignore Saac?

Reply:
Ngga, nanti tak bikinin config buat beresin ini